proton_driver/columns/datecolumn.py

```diff
@@ -34,7 +34,7 @@ def before_write_items(self, items, nulls_map=None):
                 items[i] = null_value
                 continue
 
-            if type(item) != date:
+            if item is not date:
```

Comment:
isinstance(item, date)?

Reply:
Can't use `isinstance` here,`datetime` is a instance of `date`,that make a mistake.A more accurate approach uses `type(item) is not date`,I've modified it just now.